policy/modules/system/sysnetwork.if

```diff
@@ -718,6 +718,62 @@ interface(`sysnet_signull_ifconfig',`
 	allow $1 ifconfig_t:process signull;
 ')
 
+########################################
+## <summary>
+##	Create the /run/netns directory with
+##	an automatic type transition.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`sysnet_create_netns_dirs',`
+	gen_require(`
+		type ifconfig_runtime_t;
+	')
+
+	files_runtime_filetrans($1, ifconfig_runtime_t, dir, "netns")
+')
```

Comment:
This is a filetrans interface. Something like `sysnet_runtime_filetrans_netns_dir`. This case it's not `dirs` since this is limited to one specific name.

Reply:
I use to have this as part of my `iproute` module as well. until i noticed that theres also a `/run/user/%{USERID}/netns` used by `cni`.  Thats when I decided to make that "netns" runtime location independent.